experimental/content-type-screen-effect/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "@xmtp/experimental-content-type-screen-effect",
```

Comment:
@nplasterer @jhaaaa should we indicate the experimental nature of these content types in the package name, or will putting a note in the docs be enough?

Reply:
personally i like having these explicitly called out as experimental everywhere we can, including package name.

Reply:
Ya. I can't see any harm in being more explicit here.